### PR TITLE
Support chain id for combination alerts

### PR DESCRIPTION
**Pull request by aomerk**

### CODE REVIEW

File: clients/graphql/genqlient.graphql [hunk 1]
```diff
@@ -53,11 +53,13 @@ query getAlerts($input: AlertsInput) {
                     hash
                     botId
                     timestamp
+                    chainId
```
Comment (haseebrabbani) [inline clients/graphql/genqlient.graphql:56]: does this require changes on the graphql api?

File: feeds/combiner_test.go [hunk 2]
```diff
@@ -51,14 +51,14 @@ func Test_combinerFeed_StartRange(t *testing.T) {
...
+						APIUrl: "https://api-dev.forta.network/graphql",
 					},
 				)
 				r.NoError(err)
 				cf.AddSubscription(
 					&protocol.CombinerBotSubscription{
-						BotId:   "0x5e13c2f3a97c292695b598090056ba5d52f9dcc7790bcdaa8b6cd87c1a1ebc0f",
-						AlertId: "FORTA-6",
+						BotId:   "0x2bee737433c0c8cdbd924bbb68306cfd8abcf0e46a6ce8994fa7d474361bb186",
+						// AlertId: "FORTA-6",
```
Comment (haseebrabbani) [inline feeds/combiner_test.go:61]: should we uncomment this? also, is alertId required for the subscriptions i.e. if I wanted to get all alerts from a bot, do I need to know all of its possible alert IDs?

**Review by haseebrabbani — APPROVED** [review]

File: clients/graphql/genqlient.graphql (hunk 1 above)
Comment (aomerk) [reply clients/graphql/genqlient.graphql:56]: that's already merged. Related PR:
https://github.com/forta-network/forta-alerts-api/pull/36

File: feeds/combiner_test.go (hunk 2 above)
Comment (aomerk) [reply feeds/combiner_test.go:61]: This PR prevents that behavior:
https://github.com/forta-network/forta-core-go/pull/147

**Pull request merged**